pointers/pointers.c

```diff
@@ -71,24 +68,53 @@ char *find_char(char *str, int c)
 
     Do not use the `strstr` function from the standard library.
 */
-/* char *find_string(char *haystack, char *needle) */
-/* { */
+char *find_string(char *haystack, char *needle)
+{
+
+  char *target = "delete thise";
+
+  /* int length = 0; */
+  while(*needle != '\0')
+  {
+    *needle ++;
```

Comment:
notice this is what we talked about! it still does what you want (I assume: incrementing the address that `needle` points to), but that dereference `*` is extra - it's doing nothing, and it might confuse someone else reading your code

https://en.cppreference.com/w/c/language/operator_precedence